plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHttpMetastoreConfig.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.metastore.thrift;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import io.airlift.configuration.LegacyConfig;
+import io.airlift.configuration.validation.FileExists;
+import io.airlift.units.Duration;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.NotNull;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class ThriftHttpMetastoreConfig
+{
+    public enum AuthenticationMode
+    {
+        BEARER
+    }
+
+    private AuthenticationMode authenticationMode;
+    private Optional<String> httpBearerToken = Optional.empty();
+    private Map<String, String> additionalHeaders = ImmutableMap.of();
+    private Duration readTimeout = new Duration(10, TimeUnit.SECONDS);
+    private boolean tlsEnabled;
+    private File keystorePath;
+    private String keystorePassword;
+    private File truststorePath;
+    private String trustStorePassword;
```

Comment:
These settings seem unnessary for supporting Databricks powered  HMS over HTTP.

I'd rather not have them until we find a  use case for them.

Reply:
agreed. Removed most of them